netbox/virtualization/api/nested_serializers.py

```diff
@@ -72,3 +73,12 @@ class NestedVMInterfaceSerializer(WritableNestedSerializer):
     class Meta:
         model = VMInterface
         fields = ['id', 'url', 'display', 'virtual_machine', 'name']
+
+
+class NestedVirtualDiskSerializer(WritableNestedSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name='virtualization-api:virtualdisk-detail')
+    virtual_machine = NestedVirtualMachineSerializer(read_only=True)
+
+    class Meta:
+        model = VirtualDisk
+        fields = ['id', 'url', 'virtual_machine', 'name', 'size']
```

Comment:
Nested serializers should include the `display` field, but we can omit the `size` field.

```suggestion
        fields = ['id', 'url', 'display', 'virtual_machine', 'name']
```